dev/source/docs/git-clone.rst

```diff
@@ -13,10 +13,12 @@ Clone a repository
 
 Developers should clone the main `ArduPilot repository <https://github.com/ArduPilot/ardupilot>`__ (if they simply want to download and compile the latest code) or their own :ref:`fork <git-fork>` (if they want to make changes to the source code and potentially submit changes back).
 
-Instructions for commonly used tools are below but whichever tool is used, the URL for the source repo will be required.  This can be found on the right side of the screen on each Github repository home page but in general the URL is:
+Instructions for commonly used tools are below but whichever tool is used, the URL for the source repo will be required. This can be found on the right side of the screen on each Github repository home page but in general the URL is:
 
-- ``https://github.com/ArduPilot/ardupilot.git`` for the main ardupilot repo
-- ``https://github.com/your-github-account/ardupilot`` for your :ref:`fork <git-fork>` of the ardupilot repo
+.. note:: GitHub no longer supports cloning using https. Use an ssh address to create your clone. Add an ssh key to your GitHub profile for each computer that will be used for ArduPilot development.
```

Comment:
I just cloned a new ArduPilot git repo using the  https method, so I don't believe this is true

Reply:
Was that the ardupilot master or your own repo?

Reply:
a fork of the MAVProxy repo....and I just did it from the master also

Reply:
henry@hw-ubuntu:~/Desktop$ git clone https://github.com/ArduPilot/MAVProxy.git
Cloning into 'MAVProxy'...
remote: Enumerating objects: 20250, done.
remote: Counting objects: 100% (7831/7831), done.
remote: Compressing objects: 100% (726/726), done.
remote: Total 20250 (delta 7385), reused 7122 (delta 7101), pack-reused 12419
Receiving objects: 100% (20250/20250), 66.11 MiB | 21.92 MiB/s, done.
Resolving deltas: 100% (14924/14924), done.

Reply:
That's very strange, GitHub had definitely been refusing to let me connect using https and insisted that I use ssh, which I have been doing for some time now. I wonder if something changed?

Reply:
Oh I know what it is. You will be able to clone, but not able to push changes from the clone if you use https.

Reply:
Right...that changed months ago...no more password pushs...different topic than this pr

Reply:
I think its relevant because there's really no point creating a clone of your own fork if you can't push to it, so I've made it https for ardupilot or ssh for "your fork".